x-pack/plugins/security_solution/public/management/pages/event_filter/constants.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ExceptionListType } from '../../../../../lists/common/shared_exports';
+
+export const EVENT_FILTER_LIST_ID = '.endpointEventFilterList';
+export const EVENT_FILTER_LIST_TYPE: ExceptionListType = 'endpoint_events';
+
+export const EVENT_FILTER_LIST = {
+  name: 'Endpoint Event Filter List',
+  namespace_type: 'agnostic',
+  description: 'Endpoint Event Filter List',
+  list_id: EVENT_FILTER_LIST_ID,
+  type: EVENT_FILTER_LIST_TYPE,
+};
+
+export const EXCEPTION_LIST_ITEM_URL = '/api/exception_lists/items';
+export const EXCEPTION_LIST_URL = '/api/exception_lists';
```

Comment:
These two should also be imported from https://github.com/elastic/kibana/blob/9a0c73e515f0e0c7ff88d6b06df46aa34cd8d84f/x-pack/plugins/lists/common/constants.ts#L16-L20